lib/ts/recipe/passwordless/index.ts

```diff
@@ -49,8 +49,8 @@ export default class RecipeWrapper {
      */
     static async createCode(
         input:
-            | { email: string; userContext?: any; options?: RecipeFunctionOptions }
-            | { phoneNumber: string; userContext?: any; options?: RecipeFunctionOptions }
+            | { email: string; userContext?: any; factorIds?: string[]; options?: RecipeFunctionOptions }
+            | { phoneNumber: string; userContext?: any; factorIds?: string[]; options?: RecipeFunctionOptions }
```

Comment:
why is factorIds an array of string? And why is it called factorIds instead of just `factorId: string`

Reply:
Also, will this be needed to be set during first factor as well? Cause if yes, we should make it compulsory, and type if like `otp-email` | `otp-phone` | `link-email` | `link-phone`